rel/app/env.bat.eex

```diff
@@ -9,7 +9,6 @@ set vendor_dir=!RELEASE_ROOT!\vendor\livebook-!RELEASE_VSN!
 set MIX_ARCHIVES=!vendor_dir!\archives
 set MIX_REBAR3=!vendor_dir!\rebar3
 if not defined LIVEBOOK_SHUTDOWN_ENABLED set LIVEBOOK_SHUTDOWN_ENABLED=true
-if not defined LIVEBOOK_PORT (set LIVEBOOK_PORT=0)
```

Comment:
Let’s set it to something random, 51825, or whatever.

Reply:
I set it to 32123, which does not fall into the usual ephemeral range. I guess it's unlikely enough to be taken, that I wouldn't worry about the fallback, but if you think it's worth it, that's fine too :)